kubernetes/salt/config/master.d/salt-master.conf

```diff
@@ -5,11 +5,11 @@ interface: 0.0.0.0
 event_return: mysql
 
 mysql:
-  host: '127.0.0.1'
+  # salt does not support specifying the UNIX socket location here - as a workaround,
```

Comment:
It is a bit of an oversight in the pillar. Maybe I should submit a PR to address that 😄